cpp/tools/check_resource_binding.cpp

```diff
@@ -0,0 +1,420 @@
+/**
```

Comment:
All of this string parsing will be very hard to maintain, writing it in Python might help a bit?